services/frontend/src/components/GoogleButton.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { GoogleLogin, GoogleLogout } from 'react-google-login';
+import { connect } from 'react-redux';
+import * as PropTypes from 'prop-types';
+import axios from 'axios';
+import { navigate } from 'gatsby';
+import { login, logout } from './GoogleStore';
+
+//Necessary to use the google button
+const clientID = '850674143860-haau84mtom7b06uqqhg4ei1jironoah3.apps.googleusercontent.com';
```

Comment:
If we wanna be secure we should put this in the AWS Secrets Manager and then load it in at runtime, we don't have to tho